src/components/tile-select/__internal__/accordion/accordion.test.tsx

```diff
@@ -0,0 +1,51 @@
+import React from "react";
+import { render, screen } from "@testing-library/react";
+import { act } from "react-dom/test-utils";
+import Accordion from "./accordion.component";
+import useResizeObserver from "../../../../hooks/__internal__/useResizeObserver";
+
+jest.mock("../../../../hooks/__internal__/useResizeObserver");
+
+test("the expected styles are applied to the component", () => {
+  render(<Accordion>Content</Accordion>);
+
+  const contentContainer = screen.getByTestId(
+    "tile-select-accordion-content-container"
+  );
+  const content = screen.getByTestId("tile-select-accordion-content");
+
+  expect(contentContainer).toHaveStyle({
+    backgroundColor: "var(--colorsActionMinor050)",
+  });
```

Comment:
**suggestion:** Think we should be using `toHaveStyleRule()` to test design tokens to avoid false positives.